src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.html

```diff
@@ -1,136 +1,207 @@
-                                           (cleared)="onDevicesCleared($event)">
-          </cd-osd-devices-selection-groups>
-        </fieldset>
+  <div i18n="form title|Example: Create Pool@@formTitle"
+       class="card-header"
+       *ngIf="!hideTitle">{{ action | titlecase }} {{ resource | upperFirst }}</div>
+  <div class="card-body ml-2">
+    <form name="form"
+          #formDir="ngForm"
+          [formGroup]="form"
+          novalidate>
+      <div class="accordion">
+        <div class="card">
+          <div class="card-header">
+            <h2 class="mb-0">
+              <button class="btn btn-link btn-block text-left dropdown-toggle"
+                      data-toggle="collapse"
+                      aria-label="toggle deployment options"
+                      [attr.aria-expanded]="simpleDeployment"
+                      (click)="simpleDeployment = !simpleDeployment"
+                      i18n>Deployment Options</button>
+            </h2>
+          </div>
+        </div>
+        <div class="collapse"
+             [ngClass]="{show: simpleDeployment}">
+          <div class="card-body d-flex flex-column">
+            <div class="pt-3 pb-3"
+                 *ngFor="let optionName of optionNames">
+              <div class="custom-control custom-radio custom-control-inline">
+                <input class="custom-control-input"
+                       type="radio"
+                       name="deploymentOption"
+                       [id]="optionName"
+                       [value]="optionName"
+                       formControlName="deploymentOption"
+                       (change)="emitDeploymentSelection()"
+                       [attr.disabled]="!deploymentOptions?.options[optionName].available ? true : null">
+                <label class="custom-control-label"
+                       [id]="'label_' + optionName"
+                       [for]="optionName"
+                       i18n>{{ deploymentOptions?.options[optionName].option_name }}
+                       {{ deploymentOptions.recommended_option === optionName ? "(Recommended)" : "" }}
+                  <cd-helper>
+                    <span>{{ deploymentOptions?.options[optionName].msg }}</span>
+                  </cd-helper>
+                </label>
+              </div>
+            </div>
+            <!-- @TODO: Visualize the storage used on a chart -->
+            <!-- <div class="pie-chart">
+              <h4 class="text-center">Selected Capacity</h4>
+              <h5 class="margin text-center">10 Hosts | 30 NVMes </h5>
+              <div class="char-i-contain">
+                <cd-health-pie [data]="data"
+                               [config]="rawCapacityChartConfig"
+                               [isBytesData]="true"
+                               (prepareFn)="prepareRawUsage($event[0], $event[1])">
+                </cd-health-pie>
+              </div>
+            </div> -->
```

Comment:
Let's not forget to remove this ;-)

Reply:
I've kept it for future (when we introduce the PIE chart)

Reply:
👍🏼